site/docs/components/table.mdx

```diff
@@ -1130,7 +1238,456 @@ If your table has a large amount of data, you can use the `dense` property to fo
 ```
 
 #### React code example:
-_Coming soon to hds-react!_
+```tsx
+const cols = [
+  { key: 'Paikka-ID', headerName: 'Paikka-ID' },
+  { key: 'Paikan tyyppi', headerName: 'Paikan tyyppi' },
+  { key: 'Tehtävänimike', headerName: 'Tehtävänimike' },
+  {
+    key: 'Ilmoitus vanhenee',
+    headerName: 'Ilmoitus vanhenee',
+    sortIconType: 'other' as 'other',
+    customSortCompareFunction: (a, b) => {
+      const aDate = parse(a, 'dd.MM.yyyy', new Date());
+      const bDate = parse(b, 'dd.MM.yyyy', new Date());
+
+      if (isSameDay(aDate, bDate)) {
+        return 0;
+      }
+
+      if (isBefore(aDate, bDate)) {
+        return -1;
+      }
+
+      return 1;
+    },
+  },
+  { key: 'Toimiala/liikelaitos', headerName: 'Toimiala/liikelaitos' },
+  { key: 'Työpaikka', headerName: 'Työpaikka' },
+  { key: 'Postinumero', headerName: 'Postinumero' },
+  {
+    key: 'Paikkoja',
+    headerName: 'Paikkoja',
+    sortIconType: 'other' as 'other',
+    transform: ({ Paikkoja }) => {
+      return <div style={{ textAlign: 'right' }}>{Paikkoja}</div>;
+    },
+  },
+  {
+    key: 'Haastatteluun halutaan',
+    headerName: 'Haastatteluun halutaan',
+    sortIconType: 'other' as 'other',
+    transform: (row) => {
+      return <div style={{ textAlign: 'right' }}>{row['Haastatteluun halutaan']}</div>;
+    },
+  },
+];
+
+const rows: Array<object> = workTrial;
+
+const caption = (
+  <span>
+    <b>Table 6</b>: Job trials (source: Helsinki Employment Services)
+  </span>
+);
+
+return (
+  <div style={{ maxWidth: '1200px', height: '600px' }}>
+    <Table
+      sortingEnabled
```

Comment:
Not sure if we should show sorting here, as this is only about dense tables?

Reply:
- [ ] Remove sorting from dense table doc exampl